recipes/cpp-yyjson/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+from conan.tools.microsoft import is_msvc
+import os
+
+required_conan_version = ">=1.52.0"
+
+class CppYyjsonConan(ConanFile):
+    name = "cpp-yyjson"
+    description = "Ultra-fast and intuitive C++ JSON reader/writer with yyjson backend"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/yosh-matsuda/cpp-yyjson"
+    topics = ("json", "reader", "writer", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "11",
+            "clang": "15",
+            "apple-clang": "14",
+            "Visual Studio": "17",
+            "msvc": "193",
+        }
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("yyjson/0.9.0", transitive_headers=True)
+        self.requires("fmt/10.2.1", transitive_headers=True)
+        self.requires("nameof/0.10.4", transitive_headers=True)
```

Comment:
```suggestion
        self.requires("yyjson/0.9.0")
        self.requires("fmt/10.2.1")
        self.requires("nameof/0.10.4")
```

No need to, this is already the default for `header-library` recipes!